From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: gregordinary <33880035+gregordinary@users.noreply.github.com>
Date: Wed, 22 Jul 2026 12:00:00 -0400
Subject: [PATCH] arm64: dts: rockchip: label the RK3576 GPU power domain

Add a pd_gpu label to the RK3576 GPU power domain node so boards can
attach its supply with a domain-supply property, matching the RK3588
pd_gpu/pd_npu pattern. This pairs with the pmdomain regulator support for
the RK3576 GPU domain: without the supply enabled before the domain is
powered, a deferred GPU-consumer probe races the regulator core's
disable-unused pass and the domain fails to power up.

Signed-off-by: gregordinary <33880035+gregordinary@users.noreply.github.com>
---
 arch/arm64/boot/dts/rockchip/rk3576.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3576.dtsi b/arch/arm64/boot/dts/rockchip/rk3576.dtsi
index 28175d820..d25caf5a5 100644
--- a/arch/arm64/boot/dts/rockchip/rk3576.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3576.dtsi
@@ -1084,7 +1084,7 @@ power-domain@RK3576_PD_NPU1 {
 					};
 				};
 
-				power-domain@RK3576_PD_GPU {
+				pd_gpu: power-domain@RK3576_PD_GPU {
 					reg = <RK3576_PD_GPU>;
 					clocks = <&cru CLK_GPU>, <&cru PCLK_GPU_ROOT>;
 					pm_qos = <&qos_gpu>;
-- 
2.43.0

